modules/compute/eks/requirements.txt

```diff
@@ -34,7 +34,7 @@ cattrs==22.1.0
     # via jsii
 cdk-nag==2.12.29
     # via -r requirements.in
```

Comment:
Why this change?  I think the cryptography lib was updated by the bots?

Reply:
Can you verify now?